File: sdkwork-app-sdk-typescript/src/types/user-info-vo.ts
/** 用户信息 */
export interface UserInfoVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 用户ID */
  id?: number;
  /** 用户名 */
  username?: string;
  /** 邮箱 */
  email?: string;
  /** 手机号 */
  phone?: string;
  /** 昵称 */
  nickname?: string;
  /** 头像 */
  avatar?: string;
  /** 角色 */
  role?: string;
  /** 状态 */
  status?: string;
}

File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/user_info_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class UserInfoVO:
    """用户信息"""
    created_at: str = None
    updated_at: str = None
    id: int = None
    username: str = None
    email: str = None
    phone: str = None
    nickname: str = None
    avatar: str = None
    role: str = None
    status: str = None
